airbyte-webapp/src/core/form/types.ts

```diff
@@ -2,7 +2,16 @@ import { JSONSchema7Type, JSONSchema7TypeName } from "json-schema";
 
 import { AirbyteJSONSchema } from "core/jsonSchema";
 
-interface FormItem {
+/**
+ * When turning the JSON schema into `FormBlock`s,
+ * some often used props are copied over for easy access.
+ */
+type FormRelevantJSONSchema = Pick<
+  AirbyteJSONSchema,
+  "default" | "examples" | "description" | "pattern" | "order" | "const" | "title" | "airbyte_hidden" | "enum"
+>;
+
+interface FormItem extends FormRelevantJSONSchema {
```

Comment:
Is there a reason to keep `order`, `title`, `description`, and `airbyte_hidden` in this interface, now that they are coming from `FormRelevantJSONSchema`?

Reply:
There is not, good catch.